pdns/recursordist/docs/settings.rst

```diff
@@ -879,8 +879,11 @@ If set, EDNS options in incoming queries are extracted and passed to the :func:`
 ``hint-file``
 -------------
 -  Path
+-  Default: empty
 
-If set, the root-hints are read from this file. If unset, default root hints are used.
+If set, the root-hints are read from this file. If empty, the default built-in root hints are used.
+In some special cases, processing the root hints is not needed, for example when forwarding all queries to another recursor.
+For these special cases, it is possible to disable the processing of root hints by setting the value to ``no``.
```

Comment:
I wonder if it would make sense to check whether we have a forward-recurse target for the root when that option is enabled, and to issue a loud warning (or even refuse to start) otherwise?

Reply:
Wondering as wel, don;t know if forward-recurse of root is the only one, there might be more (strange) configs I did not think of yet.